source/en/sdr.rst

```diff
@@ -565,7 +565,7 @@ For more detailed information on comparing with observations, and associated cal
 
 Following is an outline of the general steps that are done to compare modeled results against observed sediment loading data:
 
-1. Gather observed data for sediment loading at your watershed outlet of interest, process it however needed and convert to units of tons per year.
+1. Gather observed data for sediment loading at your watershed outlet of interest, process it however needed and convert to units of metric tons per year.
 
 2. Do a sensitivity analysis of the input parameters, to determine which parameters have the greatest effect on modeling results. This is most often done with LULC-based parameters (like USLE C) and "global" parameters (like *IC0* and *k*). It can also involve spatial inputs, but this is less frequently done.

```

Comment:
```suggestion
2. Do a sensitivity analysis of the input parameters to determine which have the greatest influences on modeling results. This is most often done with LULC-based parameters (e.g.: USLE C) and "global" parameters (e.g.: *IC0* and *k*). It can also involve spatial inputs, but this is less frequently done.

```